src/cljs/main/broadfcui/common/table.cljs

```diff
@@ -366,13 +366,14 @@
                 ;; Update in a single step like this to avoid multiple re-renders
                 (swap! state #(-> %
                                   (assoc :mouse-x new-mouse-x)
-                                  (assoc-in [:column-meta drag-column :width] new-width))))))))
-  (.addEventListener js/window "mousemove" (.-onMouseMoveHandler this))
-  (set! (.-onMouseUpHandler this)
+                                  (assoc-in [:column-meta drag-column :width] new-width)))))))
+        mouse-up-handler
         #(when (:dragging? @state)
-          (common/restore-text-selection (:saved-user-select-state @state))
-          (swap! state dissoc :dragging? :drag-column :mouse-x :saved-user-select-state)))
-  (.addEventListener js/window "mouseup" (.-onMouseUpHandler this)))
+           (common/restore-text-selection (:saved-user-select-state @state))
+           (swap! state dissoc :dragging? :drag-column :mouse-x :saved-user-select-state))]
+    (.addEventListener js/window "mousemove" mouse-move-handler)
+    (.addEventListener js/window "mouseup" mouse-up-handler)
+    (swap! locals assoc :mouse-move-handler mouse-move-handler :mouse-up-handler mouse-up-handler)))
```

Comment:
All of the changes here are just reorganization, changing it so these functions are stored in locals rather than attached to the object directly.